crates/bevy_pbr/src/render/light.rs

```diff
@@ -550,25 +550,22 @@ pub fn extract_lights(
             continue;
         }
 
-        // Calulate the directional light shadow map texel size using the largest x,y dimension of
+        // Calculate the directional light shadow map texel size using the scaled x,y length of
         // the orthographic projection divided by the shadow map resolution
         // NOTE: When using various PCF kernel sizes, this will need to be adjusted, according to:
         // https://catlikecoding.com/unity/tutorials/custom-srp/directional-shadows/
-        let largest_dimension = (directional_light.shadow_projection.right
-            - directional_light.shadow_projection.left)
-            .max(
-                directional_light.shadow_projection.top
-                    - directional_light.shadow_projection.bottom,
-            );
-        let directional_light_texel_size =
-            largest_dimension / directional_light_shadow_map.size as f32;
+        let directional_light_texel_size = transform.radius_vec3a(Vec3A::new(
```

Comment:
Using the radius (which is calculated as the length of the longest diagonal geometrically speaking) and scaling by sqrt(2) looks incorrect. I think also the previous code using the max dimension and sqrt(2) was incorrect as it assumed square texels which wouldn't have been the case if right - left was not the same as top - bottom.

I think using this radius and removing the factor of sqrt(2) further down would be correct.

Reply:
Oh, that makes sense. I was originally calculating a scale transform radius and just multiplying the existing texel size value by it. As per @HackerFoo I changed it to do the "proper" calculation in the transformed space, but didn't think of removing the $\sqrt{2}$ for the square's diagonal below.

Fixed.